### ISSUE TITLE
Cannot find module 'aws-xray-sdk-core/lib/utils'

### ISSUE THREAD

**Issue by drieshooghe** [narando/nest-xray#311]
Since the `aws-xray-sdk` update from 3.3.1 to 3.3.3 we get this error:
```
Error: Cannot find module 'aws-xray-sdk-core/lib/utils'
Require stack:
    - /app/node_modules/@narando/nest-xray/dist/clients/http/tracing.axios-interceptor.js
    - /app/node_modules/@narando/nest-xray/dist/clients/http/http-tracing.module.js
    - /app/node_modules/@narando/nest-xray/dist/clients/http/index.js
    - /app/node_modules/@narando/nest-xray/dist/index.js
    - /app/node_modules/@narando/nest-xray/index.js
```

Seems like the utils were moved from `aws-xray-sdk-core/dist/lib/utils` to `aws-xray-sdk-core/lib/utils`

**Comment by davad** [comment]
I got the same issue on a fresh install today.

**Comment by drieshooghe** [comment]
@MarcMogdanz this can be fixed by this PR: https://github.com/narando/nest-xray/pull/320

**Comment by MarcMogdanz** [comment]
Sorry for the delay!

I've just wrapped my head around this whole mess and implemented a fix in #273 and released under `1.5.1`. Can you verify it works for you now?

Also feel free to ping me again directly (@MarcMogdanz) that way I'll definitely see it better from now on. :)
